diff --git chrome/browser/ui/extensions/extension_dialog_utils_views.cc chrome/browser/ui/extensions/extension_dialog_utils_views.cc
index aabc0f3b09b4cf062506c92d5c749ead3aad7a23..9dc32dd1b624fc04ec88695b1be24c8407e014d8 100644
--- chrome/browser/ui/extensions/extension_dialog_utils_views.cc
+++ chrome/browser/ui/extensions/extension_dialog_utils_views.cc
@@ -89,10 +89,9 @@ void ShowDialog(BrowserWindowInterface* browser,
                 std::unique_ptr<ui::DialogModel> dialog_model) {
   ToolbarButtonProvider* toolbar_button_provider =
       BrowserView::GetBrowserViewForBrowser(browser)->toolbar_button_provider();
-  CHECK(toolbar_button_provider);
 
-  views::View* const anchor_view =
-      toolbar_button_provider->GetDefaultExtensionDialogAnchorView();
+  views::View* const anchor_view = toolbar_button_provider ?
+      toolbar_button_provider->GetDefaultExtensionDialogAnchorView() : nullptr;
   auto bubble = std::make_unique<views::BubbleDialogModelHost>(
       std::move(dialog_model), std::move(anchor_view),
       views::BubbleBorder::TOP_RIGHT);
diff --git chrome/browser/ui/views/certificate_selector.cc chrome/browser/ui/views/certificate_selector.cc
index f33e742199d99d9c78b090d8ba161cc7bf1abcac..ee6dc7ff4c2bccad0c034b980ca6bffa557de0bc 100644
--- chrome/browser/ui/views/certificate_selector.cc
+++ chrome/browser/ui/views/certificate_selector.cc
@@ -335,8 +335,13 @@ void CertificateSelector::ViewCertButtonPressed() {
   if (!cert) {
     return;
   }
+  auto native_window = web_contents_->GetTopLevelNativeWindow();
+  if (!native_window) {
+    // Not supported with CEF windowless rendering.
+    return;
+  }
   ShowCertificateViewerForClientAuth(web_contents_,
-                                     web_contents_->GetTopLevelNativeWindow(),
+                                     native_window,
                                      cert->certificate());
 }
 
