diff --git tools/gritsettings/resource_ids.spec tools/gritsettings/resource_ids.spec
index ffd47250ce5a6a75471bb65aaf3d65b550be3586..1ca7bfe4edb81d0c31fc8f8cd985458e2dc881f2 100644
--- tools/gritsettings/resource_ids.spec
+++ tools/gritsettings/resource_ids.spec
@@ -1681,6 +1681,13 @@
   # END "everything else" section.
   # Everything but chrome/, components/, content/, and ios/
 
+  "cef/libcef/resources/cef_resources.grd": {
+    "includes": [12500],
+  },
+  "cef/libcef/resources/cef_strings.grd": {
+    "messages": [12510],
+  }
+
   # Thinking about appending to the end?
   # Please read the header and find the right section above instead.
 }
