diff --git ui/views/controls/textfield/textfield.cc ui/views/controls/textfield/textfield.cc
index 0387f622e0cd3ea6fb3d5967a563f9cf0716a719..df2a4f2c6246e19fe600230cdffbc1177046688a 100644
--- ui/views/controls/textfield/textfield.cc
+++ ui/views/controls/textfield/textfield.cc
@@ -3251,6 +3251,10 @@ void Textfield::OnEnabledChanged() {
   if (GetInputMethod()) {
     GetInputMethod()->OnTextInputTypeChanged(this);
   }
+  if (GetWidget()) {
+    SetColor(GetTextColor());
+    UpdateBackgroundColor();
+  }
   UpdateDefaultBorder();
 
   // Only expose readonly if enabled. Don't overwrite the disabled restriction.
